### PR TITLE
fix #265, replaced vnu-jar with html-validate

### LINKED ISSUE #265: replace vnu-jar with html-validate
https://www.npmjs.com/package/html-validate is a Node-based alternative to the vnu-jar HTML validator. The current setup requires setting up Java on a system, which is very unintuitive for contributors and is probably also slowing CI builds down.

This would simplify the build process, which would make it easier to contribute and move towards eventually simplifying the setup so we could remove Docker and GNU Make.

Moves us closer to #264.

### PR DESCRIPTION
**Pull request by osBins**
Uninstalled the npm package `vnu-jar` and installed `html-validate` as requested in issue #265. @ryzokuken let me know if any changes are to be made!

### CODE REVIEW

**Review by jasonwilliams — APPROVED** [review]

**Comment by jasonwilliams** [comment]
There will probably need to be a script to run the html validation against how it was ran before. So you can `npm run validate`

you would need to look at what the old one was doing

**Comment by ryzokuken** [comment]
@osBins check out https://github.com/tc39/tc39.github.io/blob/master/_tasks/check-html.mjs. Since `html-validator` can be used from the CLI, you might as well just get rid of the file and replace the npm script by a single command.

**Comment by osBins** [comment]
Alright. Can you tell me how the script is to be written in package.json? 
```
"check:html": "npm exec html-validate index.html",
```  
Is this correct or is there a particular way to write the path to index.html?

**Comment by ryzokuken** [comment]
`"check:html": "html-validate _site/index.html",`

**Comment by ryzokuken** [comment]
@osBins my bad, we still need to run jekyll. 

```
"check:html": "bundle exec jekyll build && html-validate ..."
```

**Review by ryzokuken — APPROVED** [review]
LGTM. Let's see how CI proceeds.

**Comment by ryzokuken** [comment]
The new validation config is much stricter than the old one. Can we tweak the config to be functionally similar?

**Comment by osBins** [comment]
Right, looking into it.

**Comment by osBins** [comment]
I think I found it on [html-validate docs](https://html-validate.org/rules/presets.html). I need to specify a preset to configure the strictness of validation. Which one should we use?   
^Or let me know if I got it wrong.

**Comment by ryzokuken** [comment]
@osBins please use https://html-validate.org/rules/presets.html#html-validate-standard

**Comment by osBins** [comment]
Looks like the checks are still failing. Are we using the correct command for the script?  
https://html-validate.org/usage/index.html

**Comment by ryzokuken** [comment]
Yes, it runs perfectly but finds more issues with our HTML code.

Check https://app.travis-ci.com/github/tc39/tc39.github.io/builds/244771032#L562-L571 for the warnings. Perhaps try manually setting these options off too? Beyond extending the "standard" preset?

**Comment by osBins** [comment]
Had to rename a commit due to having missed out a word. The build has succeeded. Kindly check it out and let me know if anything else needs to be done.

**Review by ryzokuken — APPROVED** [review]
Great, let's move ahead! @jasonwilliams @smorimoto what do you folks think?

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "lint:js": "eslint --ext=.js,.mjs .",
     "format:js": "npm run lint:js -- --fix",
     "lint:scss": "stylelint _sass",
-    "check:html": "bundle exec jekyll build && node _tasks/check-html.mjs",
+    "check:html": "bundle exec jekyll build && html-validate _site/index.html",
```
Comment (jasonwilliams) [inline package.json]: would "lint:html" be more consistent with the above scripts? Does it have to be "check"?

Comment (ryzokuken) [reply package.json]: SGTM, let's rename it to `lint:html`. @osBins also remember to change in the CI script.

**Comment by jasonwilliams** [comment]
If this issue replaces the java validator does that mean the java dependency can be removed here? Or are you planning to remove that at another time?

**Comment by ryzokuken** [comment]
Right, good point @jasonwilliams. Let's all get rid of the Java dependency in this PR.

**Comment by osBins** [comment]
Changed the script name to `lint:html` for consistency, as pointed out by @jasonwilliams.  
Not sure about how to get rid of the Java dependency as of now. Can you help me out here @ryzokuken?

**Review by jasonwilliams — APPROVED** [review]

**Review by ryzokuken — APPROVED** [review]
LGTM.

**Pull request merged**